.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+name: "Integration test"
+# NOTE: We're just running the tests that require earthdata login here; we
+#       don't distinguish between unit and integration tests yet.
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "main"
+      - "development"
+
+
+jobs:
+  test:
+    name: "Test"
```

Comment:
```suggestion
    name: "Integration Tests"
```

For some reason this is the name GH uses in the actions interfaces... so it would mean we have a bunch of "test" options in the branch rules, which will mean I will have to forever cross reference.

Reply:
```suggestion
    name: "behind EDL test"
```

Reply:
Not sure I follow. I'm seeing this, and I was trying to avoid "Integration tests/Integration tests":

![image](https://github.com/user-attachments/assets/eb8c6c62-7f87-4f29-beb8-42e43ba96158)

Reply:
It's inconsistent. But viewed in the action tab, it only shows "Test". And when I tried to add the tests as branch rules, all I can see is like 5 versions of "test" (with varying capitals and [3.9] or [3.12]). The ones with python versions become the only names I can logically link to a specific test.

<img width="133" alt="image" src="https://github.com/user-attachments/assets/a2de6d2a-5907-414e-af07-7d2135272e4a">

Reply:
Yuck! Thanks, GitHub :laughing:

Reply:
7bb6cb5